src/systems/filecoin_markets/storage_market/storage_client.id

```diff
@@ -1,18 +1,62 @@
 import ipld "github.com/filecoin-project/specs/libraries/ipld"
-import deal "github.com/filecoin-project/specs/systems/filecoin_markets/deal"
-import piece "github.com/filecoin-project/specs/systems/filecoin_files/piece"
+import deal "github.com/filecoin-project/specs/systems/filecoin_markets/storage_market/storage_deal"
+import actor "github.com/filecoin-project/specs/systems/filecoin_vm/actor"
+import libp2p "github.com/filecoin-project/specs/libraries/libp2p"
+import addr "github.com/filecoin-project/specs/systems/filecoin_vm/actor/address"
+import block "github.com/filecoin-project/specs/systems/filecoin_blockchain/struct/block"
+
+type ClientDeal struct {
```

Comment:
Please document this, I'm not sure what it represents.

Reply:
Likewise. Name may be confusing if this is not just a deal or a proposal...?

Reply:
Agree with the above but as Hannah mentioned above, it is for client to keep track of a deal's lifecycle. Maybe rename the struct to something else?